src/test/resources/application-test.yml

```
@@ -0,0 +1,28 @@
+  access-expiration: 3600000
+  refresh-expiration: 604800000
+
+gemini:
+  prompt: gemini-test-prompt
+
+kma:
+  service-key: kma-test-service-key
+
+kakao:
+  rest-key: kakao-test-rest-key
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)